pootle/static/js/editor/app.js

```diff
@@ -376,7 +378,8 @@ PTL.editor = {
           this.category = params.category;
         }
         if ('sort' in params) {
-          this.sortBy = params.sort;
+          const { sort } = params;
+          this.sortBy = ALLOWED_SORTS.indexOf(sort) !== -1 ? sort : 'default';
```

Comment:
Just noticed that this includes `priority` sorting which might not necessarily be present in the dropdown.